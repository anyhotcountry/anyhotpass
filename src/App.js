import React from 'react';
import AnyHotPass from './AnyHotPass';
import {
  BrowserRouter as Router,
  Route
} from "react-router-dom";

const App = () => (
  <Router>
    <div className="container">
      <div className="card">
        <div className="card-body">
          <h3>AHP</h3>
          <Route path="/">
            <AnyHotPass />
          </Route>
        </div>
      </div>
    </div>
  </Router>
);

export default App;
