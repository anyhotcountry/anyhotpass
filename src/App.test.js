import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

test('App render matches snapshot', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
