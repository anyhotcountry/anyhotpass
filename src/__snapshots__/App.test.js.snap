// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="card"
    >
      <div
        class="card-body"
      >
        <h3>
          AHP
        </h3>
        <form>
          <div
            class="form-group"
          >
            <input
              class="form-control"
              placeholder="Master Password"
              required=""
              type="password"
              value=""
            />
          </div>
          <div
            class="form-group"
          >
            <input
              class="form-control"
              placeholder="example.com"
              required=""
              type="url"
              value=""
            />
          </div>
          <button
            class="btn btn-primary btn-lg btn-block "
            type="submit"
          >
            Generate
          </button>
          <div
            class="form-group mt-3"
          >
            <input
              class="form-control"
              readonly=""
              type="text"
              value=""
            />
          </div>
        </form>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
